src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -1750,33 +1752,36 @@ public static synchronized Registration register(Registration registration) {
               copyShallowFrom(registration.service, CodecUtils.fromJson(registration.getState(), Class.forName(fullTypeName)));
             }
           } catch (ClassNotFoundException classNotFoundException) {
-            log.error(String.format("Unknown service class for %s@%s: %s", registration.getName(),
-                    registration.getId(), registration.getTypeKey()), classNotFoundException);
+            log.error(String.format("Unknown service class for %s@%s: %s", registration.getName(), registration.getId(), registration.getTypeKey()), classNotFoundException);
             return null;
           }
         } else {
-          //We're registering a foreign process service. We don't need to check
-          //ForeignProcessUtils.isForeignTypeKey() because the type key is valid
-          //but is not a java class name
-
-          //Class does not exist, check if registration has empty interfaces
-          //Interfaces should always include ServiceInterface if coming from remote client
+          // We're registering a foreign process service. We don't need to check
+          // ForeignProcessUtils.isForeignTypeKey() because the type key is
+          // valid
+          // but is not a java class name
+
+          // Class does not exist, check if registration has empty interfaces
+          // Interfaces should always include ServiceInterface if coming from
+          // remote client
           if (registration.interfaces == null || registration.interfaces.isEmpty()) {
-            log.error("Unknown service type being registered, registration does not contain any " +
-                    "interfaces for proxy generation: " + registration.getTypeKey());
+            log.error("Unknown service type being registered, registration does not contain any " + "interfaces for proxy generation: " + registration.getTypeKey());

```

Comment:
And here, oddly the concatenation was left in

Reply:
auto-formatting with a large max width